src/Modules/Helper.js

```diff
@@ -2,7 +2,7 @@
 // Gets url and adds to it the cors cheat + apikey
 const getFullUrl = function(url) {
```

Comment:
Move it to APIGateway.js which I talked about earlier. He will handle all auth related stuff.